### ISSUE TITLE
ZeroDivisionError in graspologic.layouts.nooverlap

### ISSUE THREAD

**Issue by pbourke** [microsoft/graspologic#787]
## Expected Behavior
Graph layout succeeds

## Actual Behavior
ZeroDivisionError during no overlap phase

## Example Code
```
python -m graspologic.layouts --verbose True n2vumap --edge_list edgelist.csv --image_file render.png --location_file locations.csv --skip_header
```

## Full Traceback
```pytb
Traceback (most recent call last):
  File "/home/oregon/projector-docker/pythons/3.7.10/lib/python3.7/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/oregon/projector-docker/pythons/3.7.10/lib/python3.7/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/home/oregon/env3.7.10/lib/python3.7/site-packages/graspologic/layouts/__main__.py", line 316, in <module>
    main()
  File "/home/oregon/env3.7.10/lib/python3.7/site-packages/graspologic/layouts/__main__.py", line 312, in main
    args.func(args)
  File "/home/oregon/env3.7.10/lib/python3.7/site-packages/graspologic/layouts/__main__.py", line 71, in _umap
    graph, positions = auto.layout_umap(graph, max_edges=arguments.max_edges)
  File "/home/oregon/env3.7.10/lib/python3.7/site-packages/graspologic/layouts/auto.py", line 179, in layout_umap
    positions = _node_positions_from(lcc_graph, labels, points, random_seed=random_seed)
  File "/home/oregon/env3.7.10/lib/python3.7/site-packages/graspologic/layouts/auto.py", line 253, in _node_positions_from
    positions = remove_overlaps(positions)
  File "/home/oregon/env3.7.10/lib/python3.7/site-packages/graspologic/layouts/nooverlap/nooverlap.py", line 23, in remove_overlaps
    qt.layout_dense_first(first_color=None)
  File "/home/oregon/env3.7.10/lib/python3.7/site-packages/graspologic/layouts/nooverlap/_quad_tree.py", line 65, in layout_dense_first
    qn.layout_quad()
  File "/home/oregon/env3.7.10/lib/python3.7/site-packages/graspologic/layouts/nooverlap/_quad_node.py", line 635, in layout_quad
    self._do_contraction_with_given_nodes(root.nodes)
  File "/home/oregon/env3.7.10/lib/python3.7/site-packages/graspologic/layouts/nooverlap/_quad_node.py", line 1039, in _do_contraction_with_given_nodes
    len_c_new * math.sin(math.radians(angle_b_new)) / math.sin(angle_c)
ZeroDivisionError: float division by zero
```

## Your Environment
* Python version: 3.7.10
* graspologic version: 0.1.0

## Additional Details
Any other contextual information you might feel is important.

**Comment by dwaynepryce** [comment]
@bryantower

**Comment by dwaynepryce** [comment]
@bryantower what have you done

**Comment by bryantower** [comment]
I believe that is the issue that was fixed in PR 767. It should be fixed in 0.2.0. But I will verify

**Comment by dwaynepryce** [comment]
now I feel bad for teasing

**Comment by bryantower** [comment]
@pbourke, can you retry with version 0.2 or 0.3?

**Comment by pbourke** [comment]
@bryantower - yep, just updated to 0.2, which is the latest in PyPI, and will retry now. This seems to be a transient error because it succeeded on a subsequent run with 0.1.

**Comment by bryantower** [comment]
It was definitely transient on 0.1, It should be fixed on 0.2 and greater. Can we close this now?

**Comment by pbourke** [comment]
yeah this did not happen again - closing
